fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/RowArrayType.java

```diff
@@ -39,6 +39,11 @@ public void apply(IDatabaseAdapter target) {
         target.createArrType(getSchemaName(), getObjectName(), rowTypeName, arraySize);
     }
 
+    @Override
+    public void apply(Integer priorVersion, IDatabaseAdapter target) {
```

Comment:
We should only create this one time. I actually don't even think this is in our schema today.

Reply:
yeah, i think this was something we used in our old (DSTU2) schema.  it got dropped when we moved to a dynamic statement because I guess those don't support array types or something (Robin knows the details).
for now, I've just added the same logic that I added for other things that shouldn't get updated:
```
        if (priorVersion != null && priorVersion > 0 && this.version > priorVersion) {
            throw new UnsupportedOperationException("Upgrading row array types is not supported");
        }
```